src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -903,13 +903,13 @@ public partial class DataView : System.ComponentModel.MarshalByValueComponent, S
         [System.ComponentModel.DefaultValueAttribute(System.Data.DataViewRowState.CurrentRows)]
         public System.Data.DataViewRowState RowStateFilter { get { throw null; } set { } }
         [System.ComponentModel.DefaultValueAttribute("")]
-        public string? Sort { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
+        public string Sort { get { throw null; } set { } }
         bool System.Collections.ICollection.IsSynchronized { get { throw null; } }
         object System.Collections.ICollection.SyncRoot { get { throw null; } }
         bool System.Collections.IList.IsFixedSize { get { throw null; } }
         bool System.Collections.IList.IsReadOnly { get { throw null; } }
         object? System.Collections.IList.this[int recordIndex] { get { throw null; } set { } }
-#nullable disable
```

Comment:
Did @roji or someone else finish going through all of these APIs to annotate them appropriately?

Reply:
I don't believe so, I'll revert this change

Reply:
> I don't believe so

The next obvious question then is... can we?  Leaving some APIs as oblivious is unexpected.

Reply:
I believe these depend on TypeConverters being annotated correctly. However that will be for 6.0.0, I will annotate these as part of that if we agree we shouldn't annotate them now.

Reply:
@stephentoub theoretically we could but:
- @roji might have more context why this wasn't annotated in the first place
- I believe we're supposed to not annotate any more new code even if we left parts of the assembly not annotated and make sure we have time to flush any bugs related although this arguably could be treated as a bug

Reply:
> I believe these depend on TypeConverters being annotated correctly.

Ok

> I believe we're supposed to not annotate any more new code even if we left parts of the assembly not annotated and make sure we have time to flush any bugs related although this arguably could be treated as a bug

Ok, but we should at least make sure this is tracked in some way. I didn't see any mention in Jeff's summary of work still to do that suggests we're tracking still unannotated portions of System.Data.

Reply:
I've updated https://github.com/dotnet/runtime/issues/41720 to have this info

Reply:
> Ok, but we should at least make sure this is tracked in some way. I didn't see any mention in @jeffhandley's summary of work still to do that suggests we're tracking still unannotated portions of System.Data. (maybe I missed it?)

@roji and @jeffhandley let me know that was part of annotating TypeConverters, so I was considering that as part of my task.

Reply:
I thought these were disabled from XML dependency and by my understanding XML dependency for System.Data.Common is resolved, so its safe to remove it, thanks for reverting

Reply:
Yeah, I can confirm - when annotating System.Data.Common (and others) I deliberately left oblivious any API that depends on TypeConverters (as here) and XML. Those should be the only reason anything should still be oblivious in System.Data.Common (at least under ref).